### ISSUE TITLE
Error at java11 when new drive cause by module

### ISSUE THREAD

**Issue by baha2046** [SeleniumHQ/selenium#7676]
## 🐛 Bug Report

WebDriver driver = new ChromeDriver();

produce an exception at runtime:

```
Caused by: java.util.ServiceConfigurationError: org.openqa.selenium.remote.session.CapabilitiesFilter: module org.seleniumhq.selenium.remote_driver does not declare `uses`
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:588)
	at java.base/java.util.ServiceLoader.checkCaller(ServiceLoader.java:574)
	at java.base/java.util.ServiceLoader.<init>(ServiceLoader.java:503)
	at java.base/java.util.ServiceLoader.load(ServiceLoader.java:1691)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.NewSessionPayload.<init>(NewSessionPayload.java:125)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.NewSessionPayload.create(NewSessionPayload.java:101)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.NewSessionPayload.create(NewSessionPayload.java:94)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:69)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:139)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:576)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:210)
	at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:130)
	at org.seleniumhq.selenium.chromium_driver/org.openqa.selenium.chromium.ChromiumDriver.<init>(ChromiumDriver.java:75)
	at org.seleniumhq.selenium.chrome_driver/org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:163)
	at org.seleniumhq.selenium.chrome_driver/org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:150)
	at org.seleniumhq.selenium.chrome_driver/org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:105)
```

## To Reproduce

java11 project with maven,
>        <dependency>
>            <groupId>org.seleniumhq.selenium</groupId>
>            <artifactId>selenium-java</artifactId>
>            <version>4.0.0-alpha-3</version>
>        </dependency>

module-info.java
```    
    requires org.seleniumhq.selenium.api;
    requires org.seleniumhq.selenium.chrome_driver;
```    

when running to following code
```    
     WebDriver driver = new ChromeDriver();
```

>Starting ChromeDriver 77.0.3865.40 (f484704e052e0b556f8030b65b953dce96503217-refs/branch-heads/3865@{#442}) on port 44634
Only local connections are allowed.
Please protect ports used by ChromeDriver and related test frameworks to prevent access by malicious code.

Then a Exception since the service cannot start.
     

## Expected behavior

Exception is from java.base/java.util.ServiceLoader.checkCaller(ServiceLoader.java:574)

```
        // If the caller is in a named module then it should "uses" the
        // service type
        if (!callerModule.canUse(svc)) {
            fail(svc, callerModule + " does not declare `uses`");
        }
```

The problem is at org.seleniumhq.selenium.remote_driver/org.openqa.selenium.remote.NewSessionPayload.<init>(NewSessionPayload.java:125)

```    
        ServiceLoader var10000 = ServiceLoader.load(CapabilitiesFilter.class);
```    

but java.util.ServiceLoader unable to load class which haven't define uses at module-info.java.
Adding 
```    
    uses org.openqa.selenium.remote.session.CapabilitiesFilter;
```    
to module-info of org.seleniumhq.selenium.remote_driver should solve the problem.

## Environment

OS: Windows 10
Browser: 
Browser version: 
Browser Driver version: 
Language Bindings version: Java 4.0.0-alpha-3 
Selenium Grid version (if applicable):

**Comment by cgoldberg** [comment]
please show the full stacktrace

**Comment by baha2046** [comment]
sorry, i already switch back to version 3.141.59.
and i don't think the full stack is useful since the trace below 
'''
at org.seleniumhq.selenium.chrome_driver/org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:105)
'''
 just shows which part of my programming is calling the webdrive.
And the cause of the error is known.

**Comment by barancev** [comment]
NP, I understood the report and know what causes the issue, thank you for the report!

**Comment by baha2046** [comment]
This is the module-info.class of 3.141.59, these "uses" is missing at 4.0.0
![111](https://user-images.githubusercontent.com/44514585/66674637-51085800-ec9e-11e9-923a-f1331c6836ee.jpg)

**Comment by barancev** [comment]
Fixed by d26fe1f642a99f2b85e16265ef540f52c6612ff9

**Comment by barancev** [comment]
Forgot to close it. To be released in 4.0.0-alpha-4
